LICENSE

```diff
@@ -178,6 +178,14 @@
 
 --------------------------------------------------------------------------------
 
+This product includes code from Apache Avro.
```

Comment:
This includes logical types tests from Avro, updated to use Parquet as the underlying format.